### PR TITLE
Update consistency05

### PR DESCRIPTION
**Pull request by matsduf**
Clarified the DNS lookup. No change of logic. Some editorial updates.

### CODE REVIEW

**Comment by sandoche2k** [comment]
If you are using the new "Methods", then it should be reviewed.. isn't it?

**Comment by matsduf** [comment]
@sandoche2k, it is not referring to the new version of Methods, is it?

File: docs/specifications/tests/Consistency-TP/consistency05.md [hunk 1]
```diff
@@ -169,43 +166,42 @@ in [RFC 7719], section 6, page 15.
...
+The term "send" (to an IP address) is used when a DNS query is sent to
+a specific name server.
 
-[BASIC01]: ../Basic-TP/basic01.md
+The term "DNS Lookup" is used when a recursive lookup is used, though
+any changes to the DNS tree introduced by an undelegated test must be
+respected.
 
+
+[name server requirement]: https://www.iana.org/help/nameserver-requirements
```
Comment (mattias-p) [inline docs/specifications/tests/Consistency-TP/consistency05.md]: This list is getting long. Please consider sorting it alphabetically.

File: docs/specifications/tests/Consistency-TP/consistency05.md [hunk 2]
```diff
@@ -169,43 +166,35 @@ in [RFC 7719], section 6, page 15.
...
-
-[CHILD_NS_FAILED]: #outcomes
-
-[NO_RESPONSE]: #outcomes
-
-[CHILD_ZONE_LAME]: #outcomes
-
-[IN_BAILIWICK_ADDR_MISMATCH]: #outcomes
-
+When the term "using Method" is used, names and IP addresses are fetch
```
Comment (mattias-p) [inline docs/specifications/tests/Consistency-TP/consistency05.md]: fetch -> fetched

Comment (matsduf) [reply docs/specifications/tests/Consistency-TP/consistency05.md]: Fixed.

**Comment by matsduf** [comment]
@sandoche2k, you are fine with this, aren't you?

**Pull request merged**